tillerinobot/src/main/java/tillerino/tillerinobot/lang/Italiano.java

```diff
@@ -9,13 +9,13 @@
 import tillerino.tillerinobot.CommandHandler.Action;
 import tillerino.tillerinobot.CommandHandler.Message;
 import tillerino.tillerinobot.CommandHandler.Response;
-
 /**
  * Italian Language implementation. Thanks go to
  * https://osu.ppy.sh/u/marcostudios and https://osu.ppy.sh/u/Howl for this.
  */
-public class Italiano implements Language {
- 
+public class Default implements Language {
```

Comment:
sorry, i copy pasted the whole this to avoid typos but i forgot to change the class